requirements.txt

```diff
@@ -15,3 +15,5 @@ jira
 simplejson
 pyvirtualdisplay
 Faker
+tftpy
+shutil
```

Comment:
These are not used in this repo. We will not add them.

Reply:
Also shutil is not something you pip install.

Reply:
got it, thx.